docs/source/experiment.rst

```diff
@@ -6,14 +6,14 @@ Experiment
 There are two main abstractions used in the ``prompttools`` library: Experiments and Harnesses.
 Occasionally, you may want to use a harness, because it abstracts away more details.
 
-An experiment is a low level abstraction that takes the cartesian product of possible inputs to
+An experiment is a low level abstraction that takes the Cartesian product of possible inputs to
 an LLM API. For example, the ``OpenAIChatExperiment`` accepts lists of inputs for each parameter
 of the OpenAI Chat Completion API. Then, it constructs and asynchronously executes requests
 using those potential inputs. An example of using experiment is `here <https://github.com/hegelai/prompttools/blob/main/examples/notebooks/BasicExperiment.ipynb>`_.
 
 There are two ways to initialize an experiment:
 
-  1. Wrap your parameters in ``list``s and pass them into the ``__init__`` method. See each class's
+  1. Wrap your parameters in ``list`` s and pass them into the ``__init__`` method. See each class's
```

Comment:
Why are you adding a space here? The intention is to use "`list`s" as a plural of `list`.

Reply:
Sorry, I should have put that picture in the OP! 😄

Reply:
Ah, thanks for catching that! How does it render with the space?

Reply:
Building it locally, I get this:
<img width="798" alt="image" src="https://github.com/hegelai/prompttools/assets/141435510/e70c7dc9-3966-48d6-8fc6-3cb782e673c7">
It looks like there's still a space there...

Reply:
Looks like escaping that space fixes the problem: 
<img width="813" alt="image" src="https://github.com/hegelai/prompttools/assets/141435510/aa981249-5099-4213-8e58-17ed94036f64">

Reply:
Awesome, thanks for doing this!